lsix

```diff
@@ -191,6 +191,8 @@ main() {
 	readarray -t < <(printf "%s\n" "$@" | sort)
 
 	# Only show first frame of animated GIFs if filename not specified.
+  # SC2068 (error): Double quote array expansions to avoid re-splitting elements.
+  # shellcheck disable=SC2068
```

Comment:
Oog. That's ugly. lsix is supposed to be a simple script that people can easily hack upon without all the extra cognitive load of these sort of distractions.

How about just adding the GitHub Action so it can run as an advisory linter but leave all these shellcheck comments out of the code?

Reply:
These comments are neccessary for `shellcheck` so it won't complain about a specific error or warning in the next line. So for bash-hackers it should not confuse or distract them - but a failing `shellcheck` GitHub Action might confuse 🤔  
I tried to fix this as mentioned/hinted on https://github.com/koalaman/shellcheck/wiki/SC2068 - but then `lsix` failed doing the right thing 😞 So I left `shellcheck` intentionally ignore this because it works/does the right thing.

Reply:
I added a comment that should help other users understand why these shellcheck-lines are neccessary. And I fixed `(cd "$arg"; $lsix)` to `(cd "$arg" && $lsix)` - Do you @hackerb9 agree to that?